src/main/config/config.c

```diff
@@ -832,6 +832,10 @@ void activateConfig(void)
 
 void validateAndFixConfig(void)
 {
+    if (motorConfigToUse->motorPwmProtocol == PWM_TYPE_BRUSHED && motorConfigToUse->mincommand < 1000) {
```

Comment:
This function should probably check for values within `masterConfig.resetEscAndServoConfig`